app/qml/form/FeatureFormPage.qml

```diff
@@ -234,6 +241,19 @@ Item {
         }
       }
 
+      AutoHideBanner {
+        id: editingFailedBanner
+
+        width: featureForm.width - 16
```

Comment:
Let's not use random constants in code (Look at file `InputStyle.qml`,  all constants are defined there). 
Moreover, pixel values needs to be multiplied by `* __dp`.

```suggestion
        width: featureForm.width - InputStyle.innerFieldMargin
```
or
```suggestion
        width: featureForm.width - InputStyle.smallGap
```